### ISSUE TITLE
Not  working on windows

### ISSUE THREAD

**Issue by iwxfer** [codediodeio/angular-firestarter#42]
```
> ng serve --aot

** NG Live Development Server is listening on localhost:4200, open your browser on http://localhost:4200/ **
Date: 2017-12-28T18:41:03.992Z
Hash: de590bbfd9f3830769d5
Time: 3419ms
chunk {inline} inline.bundle.js (inline) 5.79 kB [entry] [rendered]
chunk {main} main.bundle.js (main) 2.93 kB [initial] [rendered]
chunk {polyfills} polyfills.bundle.js (polyfills) 636 bytes [initial] [rendered]
chunk {styles} styles.bundle.js (styles) 41.3 kB [initial] [rendered]
chunk {vendor} vendor.bundle.js (vendor) 852 kB [initial] [rendered]

ERROR in TypeError: Cannot read property 'length' of undefined
    at createSourceFile (D:\dev\zor\angular\node_modules\typescript\lib\typescript.js:16640:109)
    at parseSourceFileWorker (D:\dev\zor\angular\node_modules\typescript\lib\typescript.js:16572:26)
    at Object.parseSourceFile (D:\dev\zor\angular\node_modules\typescript\lib\typescript.js:16521:26)
    at Object.createSourceFile (D:\dev\zor\angular\node_modules\typescript\lib\typescript.js:16371:29)
    at VirtualFileStats.getSourceFile (D:\dev\zor\angular\node_modules\@ngtools\webpack\src\compiler_host.js:67:35)
    at WebpackCompilerHost.getSourceFile (D:\dev\zor\angular\node_modules\@ngtools\webpack\src\compiler_host.js:207:38)
    at TsCompilerAotCompilerTypeCheckHostAdapter.getOriginalSourceFile (D:\dev\zor\angular\node_modules\@angular\compiler-cli\src\transformers\compiler_host.js:226:31)
    at TsCompilerAotCompilerTypeCheckHostAdapter.getSourceFile (D:\dev\zor\angular\node_modules\@angular\compiler-cli\src\transformers\compiler_host.js:318:23)
    at findSourceFile (D:\dev\zor\angular\node_modules\typescript\lib\typescript.js:70246:29)
    at processImportedModules (D:\dev\zor\angular\node_modules\typescript\lib\typescript.js:70393:25)
    at findSourceFile (D:\dev\zor\angular\node_modules\typescript\lib\typescript.js:70274:17)
    at processImportedModules (D:\dev\zor\angular\node_modules\typescript\lib\typescript.js:70393:25)
    at findSourceFile (D:\dev\zor\angular\node_modules\typescript\lib\typescript.js:70274:17)
    at args (D:\dev\zor\angular\node_modules\typescript\lib\typescript.js:70200:85)
    at getSourceFileFromReferenceWorker (D:\dev\zor\angular\node_modules\typescript\lib\typescript.js:70173:34)
    at processSourceFile (D:\dev\zor\angular\node_modules\typescript\lib\typescript.js:70200:13)

webpack: Failed to compile.
```
Any ideas?

**Comment by Mayankjh** [comment]
add environment.prod.ts and environment.ts files to the src/environment folder.

**Comment by codediodeio** [comment]
Fixed by #44 Thank you @Mayankjh
